Cargo.toml

```diff
@@ -12,7 +12,7 @@ keywords = ["gui", "ui", "graphics", "interface", "widgets"]
 categories = ["gui"]
 
 [features]
-default = ["dyrend", "winit"]
+default = ["softbuffer", "winit", "a11y"]
```

Comment:
Why is this changing `dyrend` to `softbuffer`?

Reply:
That might just be a typo :sweat_smile: